docs/release-notes/v2.0.rst

```diff
@@ -1,7 +1,224 @@
+.. include:: ../references.txt
+
+
 .. _gammapy_2p0_release:
 
-2.0 (unreleased)
+2.0 (August XX, 2025)
 ----------------
 
 
-- No changes yet
+- Released August XX, 2025
+- 25 contributors
+- 235 (XX)  pull requests since v1.3 (not all listed below)
+- 100 (XX) closed issues
+
+Summary
+~~~~~~~
+
+This release introduces a number of performance improvements, new features and bug fixes.
+Improved support is provided for Fermi-LAT analysis, and full analysis for VERITAS telescope is supported.
+Bayesian inference with nester sampling is now supported.
+There has been significant code cleanup, specially with respect to the documentation and the tutorial structure has
+been cleaned up. cython code for computation of fit statistics is now replaced by jit code.
+Dependence on `lru_cache` is removed.
```

Comment:
```suggestion
been cleaned up.
```
I would not mention cython and lru_cache here but only on the details. And cython is still the default using jit is optional.